src/shared/format-table.h

```diff
@@ -88,7 +88,29 @@ int table_add_cell_full(Table *t, TableCell **ret_cell, TableDataType type, cons
 static inline int table_add_cell(Table *t, TableCell **ret_cell, TableDataType type, const void *data) {
         return table_add_cell_full(t, ret_cell, type, data, SIZE_MAX, SIZE_MAX, UINT_MAX, UINT_MAX, UINT_MAX);
 }
-int table_add_cell_stringf(Table *t, TableCell **ret_cell, const char *format, ...) _printf_(3, 4);
+int table_add_cellv(Table *t, TableCell **ret_cell, TableDataType type, const char *format, va_list ap) _printf_(4, 0);
+_printf_(4, 5) static inline int table_add_cellf(Table *t, TableCell **ret_cell, TableDataType type, const char *format, ...)
+{
```

Comment:
Coding style: please move `{` at the end of the previous line.

Reply:
I removed the inlining completely, by changing `table_add_cell_stringf` into a macro that expands to `table_add_cellf`. Which is really what I wanted to do from the beginning, just didn't think about it...